index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Leo's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div><a href="index.html">Leo</a></div>
+        <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+    </nav>
```

Comment:
- [ ] Dear @popey17, For sematic reasons, this section should wrapped inside a `header` tag, Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
- [ ] I would also suggest wrapping only the hamburger button with the `nav` tag. This is because your navigation are going to be part of this icon.